src/coreclr/src/jit/instrsarm64.h

```diff
@@ -1221,6 +1221,12 @@ INST1(umaxv,   "umaxv",  0, 0, IF_DV_2M,  0x2E30A800)
 INST1(uminv,   "uminv",  0, 0, IF_DV_2M,  0x2E31A800)
                                    //  uminv   Vd,Vn                DV_2M  0Q101110XX110001 101010nnnnnddddd   2E31 A800   Vd,Vn      (vector)
 
+INST1(trn1,    "trn1",   0, 0, IF_DV_3A,  0x0E002800)
+                                   //  zip1    Vd,Vn,Vm             DV_3A  0Q001110XX0mmmmm 001010nnnnnddddd   0E00 2800   Vd,Vn,Vm  (vector)
```

Comment:
Shouldn't the comment be about `trn1` and not `zip1` instruction?

Reply:
Yes sorry, had somehow missed updating that..